Form-Controls/index.html

```diff
@@ -14,14 +14,49 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form id="tshirt-order-form">
+      <div>
+        <label for="name"></label> Name:</label>
+        <input type="text" id="name" name="name" required pattern="[A-Za-z\s]+" title="Please enter a valid name (letters and spaces only).">
+      </div>
+      <div>
+        <label for="email"> Email:</label>
+        <input type="email" id="email" name="email" required>
+      </div>
+
+    <div>
+      <label for="color">T-shirt Color:</label>
+      <select id="color" name="color" required> 
+        <option value="">Select a color</option>
+        <option value="red">Red</option>
+        <option value="blue">Blue</option>
+        <option value="green">Green</option>
+      </select>
+    </div>
+     <div>
+      <label for="size">T-shirt Size:</label>
+      <select id="size" name="size" required>
+        <option value="">Select a size</option>
+        <option value="XS">XS</option>
+        <option value="S">S</option>
+        <option value="M">M</option>
+        <option value="L">L</option>
+        <option value="XL">XL</option>
+        <option value="XXL">XXL</option>
+      </select>
+    </div>
+    <div>
+      <label for="delivery">Delivery Date:</label>
+      <input type="date" id="delivery" name="delivery" required min="2024-10-20" max="2024-11-20" required>
```

Comment:
Date range is interesting, as the minimum date does not move in realtime  - this is where Javascript would come in